.github/pages/make_switcher.py

```diff
@@ -64,7 +64,7 @@ def write_json(path: Path, repository: str, versions: str):
     ]
     text = json.dumps(struct, indent=2)
     print(f"JSON switcher:\n{text}")
-    path.write_text(text)
+    path.write_text(text, errors="ignore", encoding="utf-8")
```

Comment:
Please could you make this change (and the one to the action if needed) to the skeleton repo rather than here

Reply:
Change is in https://github.com/DiamondLightSource/python3-pip-skeleton/pull/143, I cherry-picked it here.